event.go

```diff
@@ -196,14 +196,28 @@ func (p *eventSinkMgr) doWhenBackdropChanged(name string, wait bool) {
 }
 
 // -------------------------------------------------------------------------------------
+type IEventSinks interface {
+	OnAnyKey(onKey func(key Key))
+	OnBackdrop__0(onBackdrop func(name string))
+	OnBackdrop__1(name string, onBackdrop func())
+	OnClick(onClick func())
+	OnKey__0(key Key, onKey func())
+	OnKey__1(keys []Key, onKey func(Key))
+	OnKey__2(keys []Key, onKey func())
+	OnMsg__0(onMsg func(msg string, data interface{}))
+	OnMsg__1(msg string, onMsg func())
+	OnStart(onStart func())
+	Stop(kind StopKind)
+}
 
 type eventSinks struct {
+	IEventSinks
```

Comment:
Why is it necessary to have the IEventSinks interface as a property? It seems pointless.

Reply:
This code will be removed.